catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -692,4 +711,10 @@ public InputStream openStream() throws IOException {
             return supplier.get();
         }
     }
+
+    private List<String> getWritableSources() {
```

Comment:
Also just a nit, can we keep the static class at the bottom and move this up next to the other private methods?